updates readme (#421)
bigcommerce/docs: guides/create-a-site-preview.md

# How to create a site preview 

Creating a site preview is a straightforward process that allows you to see how your doc appears before it’s submitted.

> In this doc you’ll learn how to clone the developer-center repo to run a localhost: 3000 to preview IDE  (e.g., VS Code) changes, and how to use a static site preview generator. 

Reviewer: ```suggestion
Learn how to clone the developer-center repo to run a localhost: 3000 to preview real-time IDE (e.g., VS Code) changes.
```
